source-manager/src/main/java/com/jointrivial/sourcemanager/nordigen/service/NordigenConnectionIdService.java

```diff
@@ -0,0 +1,17 @@
+package com.jointrivial.sourcemanager.nordigen.service;
+
+import com.jointrivial.sourcemanager.nordigen.model.service.NordigenConnectionIdServiceModel;
+import com.jointrivial.sourcemanager.nordigen.model.service.RequisitionServiceModel;
+import com.jointrivial.sourcemanager.nordigen.model.view.AuthorizationLinkViewModel;
+import org.springframework.http.HttpStatus;
+
+import java.io.IOException;
+
+public interface NordigenConnectionIdService {
+
+    AuthorizationLinkViewModel getAuthorizationLink(String userToken, String bankId, String bankName, String redirectLinkPrefix) throws IOException, InterruptedException;
+
+    NordigenConnectionIdServiceModel getSourceIdentifierByReferenceId(String referenceId);
```

Comment:
This method is completely unneccesary. To get the bankName later, just add another method in the repository.

Reply:
Make two calls instead of one is unnecessary